recipes/gencove/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "2.4.3" %}
+{% set sha256 = "14d16e455142d798f680fa18b1d39377fbb0fea1d69f174bcb1b114038de5286" %}
+
+package:
+  name: gencove
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/gncv/gencove-cli/archive/refs/tags/{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  noarch: python
+  number: 0
+  script: '$PYTHON -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv '
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - python-dateutil >=2.2.0
+    - six >=1.5
+    - click >=7.0
+    - requests >=2.19.1
+    - boto3 >=1.17.97
+    - progressbar2 ==3.55.0
+    - backoff ==1.11.0
+    - pydantic ==1.9.2
+    - pytest-runner
+
+test:
+  commands:
+    - gencove upload -h
```

Comment:
what about changing test to `gencove --help`?

Reply:
```suggestion
    - gencove --help
```